addon/components/bs-modal.js

```diff
@@ -137,6 +117,25 @@ export default class Modal extends Component {
   @tracked
   paddingRight;
 
+  /**
+   * Visibility of the modal. Toggle to show/hide with CSS transitions.
+   *
+   * When the modal is closed by user interaction this property will not update by using two-way bindings in order
+   * to follow DDAU best practices. If you want to react to such changes, subscribe to the `onHide` action
+   *
+   * @property open
+   * @type boolean
+   * @default true
+   * @public
+   */
+
+  /**
+   * @property open
+   * @private
+   */
```

Comment:
```suggestion
```

This JSDoc block should be removed, as it is wrongly stating that `@open` is private, and is in direct conflict with the doc block (referring to the same `@property open`) just a few lines above.

Reply:
Good point! I've removed it now.